Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml

```diff
@@ -231,25 +227,18 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="WoodLog"]/statBases/SharpDamageMultiplier</xpath>
-		<value>
-			<SharpDamageMultiplier>0.7</SharpDamageMultiplier>
-		</value>
-	</Operation>
-
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/ThingDef[defName="WoodLog"]/statBases/BluntDamageMultiplier</xpath>
 		<value>
-			<BluntDamageMultiplier>0.5</BluntDamageMultiplier>
+			<BluntDamageMultiplier>0.8</BluntDamageMultiplier>
```

Comment:
Given that we're making wood sharp damage significantly worse by putting it back to the vanilla sharp damage factor (0.4), we might as well just get rid of this patch and let it go back to the vanilla blunt damage multi of 0.9.